test/integration/gql_api_config_test.lua

```diff
@@ -0,0 +1,190 @@
+local fio = require('fio')
```

Comment:
How to delete a config section? Imagine that I want to delete a file from the GUI and then apply changes

Reply:
You need to send grapqhl request
```graphql
mutation delete_section($sections: [ConfigSectionInput!]) {
          cluster {
                apply_config(sections: $sections) {...}
          }
}
```

Where ```ConfigSectionInput``` contains filename (is a section name in clusterwide config) and content = nil